modules/deploy/pages/deployment-option/cloud/gcp-private-service-connect.adoc

```diff
@@ -0,0 +1,379 @@
+= Configure GCP Private Service Connect for BYOC
+:description: Set up GCP Private Service Connect to securely access Redpanda Cloud. 
+:page-cloud: true
+
+include::shared:partial$feature-flag.adoc[]
+
+Redpanda GCP Private Service Connect (PSC) provides secure access to Redpanda Cloud from your own VPC. Traffic over PSC does not go through the public internet because a PSC connection is treated as its own private GCP service. While your VPC has access to the Redpanda VPC, Redpanda cannot access your VPC.
+
+Consider using PSC if you have multiple VPCs and could benefit from a more simplified approach to network management:
+
+* PSC allows overlapping CIDR ranges in VPC networks.
```

Comment:
Maybe link CIDR ranges to https://deploy-preview-346--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/cidr-ranges/#what-are-cidrs?